src/test/java/org/apache/commons/math4/analysis/integration/MidPointIntegratorTest.java

```diff
@@ -48,8 +67,9 @@ public void testLowAccuracy() {
         double expected = -3697001.0 / 48.0;
         double tolerance = FastMath.abs(expected * integrator.getRelativeAccuracy());
         double result = integrator.integrate(Integer.MAX_VALUE, f, min, max);
-        Assert.assertTrue(integrator.getEvaluations() < Integer.MAX_VALUE / 2);
+        Assert.assertTrue(integrator.getEvaluations() < Integer.MAX_VALUE / 3);
```

Comment:
This test might be redundant now that we know exactly how many evaluations we should have, as a function of iterations.